### PR TITLE
feat: amplitude swap fixes

### PR DESCRIPTION
**Pull request by lynnshaoyu**
1) change "Swap Submitted" to "Swap Submit Button Clicked"
2) Add 'Swap Signed' event when transaction hash is produced after swap approve signing. 
Screen Shot 2022-09-14 at 3 40 23 PM

Notion doc w/ details: https://www.notion.so/uniswaplabs/Amplitude-Router-Discrepancy-60dfd67b736c44ec85c4d5021d038466

### CODE REVIEW

File: src/state/transactions/updater.tsx [hunk 1]
```diff
@@ -78,19 +40,7 @@ export default function Updater() {
           },
         })
       )
-      const tx = transactions[chainId]?.[hash]
 
-      if (tx.info.type === TransactionType.SWAP && trade) {
-        sendAnalyticsEvent(
```
Comment (cmcewen) [inline src/state/transactions/updater.tsx:84]: why remove this? it does seem useful to know whether the browser was open right?

Comment (lynnshaoyu) [reply src/state/transactions/updater.tsx:84]: i think we agreed to remove it: https://uniswapteam.slack.com/archives/C03JAKVATU6/p1663003407549829?thread_ts=1662855832.386199&cid=C03JAKVATU6

Comment (lynnshaoyu) [reply src/state/transactions/updater.tsx:84]: keeping it creates confusion because this gives an incomplete picture

Comment (lynnshaoyu) [reply src/state/transactions/updater.tsx]: added back as per slack discussion

**Review by cmcewen — APPROVED** [review]

**Pull request merged**